buildSrc/src/main/groovy/airbyte-docker.gradle

```
@@ -194,7 +194,13 @@ class AirbyteDockerPlugin implements Plugin<Project> {
     }
 
     static def createTask(Project project, String taskName, String dockerFile) {
-        if (project.file(dockerFile).exists()) {
+        // airbyte-ci only supports actual connectors, so exclude all the base modules
+        if (project.name.startsWith('source-') || project.name.startsWith('destination-')) {
+            project.task(taskName, type: Exec) {
+                workingDir project.rootProject.projectDir
+                commandLine "airbyte-ci", "connectors", "--name", project.name, "build"
```

The current `airbyte-ci` call starts a terminal UI which won't play very well with execution from gradle
```suggestion
                commandLine "airbyte-ci", "--no-tui", "connectors", "--name", project.name, "build"
```